sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -64,8 +64,43 @@ APIs for using IotHub v2 jobs
 
 <details><summary><b>Messages</b></summary>
 Feedback messages, sending cloud to device messages (missing from current swagger), and purging cloud to device message queue
-```csharp
 
+```csharp
+public class Messages
+{
+    /// <summary>
+    /// Retrieve feedback notification for cloud to device messages.
+    /// </summary>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>TODO: Swagger defines the response as the Http response returned from the service (and not the feedback message).</returns>
+    public virtual async Task<Response> GetCloudToDeviceMessageFeedbackAsync(CancellationToken cancellationToken = default) { }
```

Comment:
Including the "CloudToDeviceMessage" part makes each method a mouthful.

Also, I don't know what "Feedback" is supposed to mean. The method name as a whole is hard to understand.

Might not be much you can do about it, but perhaps we can brainstorm.

Reply:
I agree, the c2d message feedback flow is not very straight-forward and easy to use. We can go over the context of the flow in today's meeting and hopefully come up with a better name for this.